### DISCUSSION TITLE
Failed to connect to github.com port 443: No route to host

### DISCUSSION THREAD

**Discussion by s0rcier911** [pocopico/tinycore-redpill#350] [Q&A]
running vmware workstation pro 17
downloaded tinycore-redpill.v0.9.4.9.vmdk -- latest

tc@box:~$ ifconfig

eth0      Link encap:Ethernet  HWaddr 00:0C:29:20:60:E6
          inet addr:10.1.1.201  Bcast:10.1.1.255  Mask:255.255.255.0
          UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1
          RX packets:1054 errors:0 dropped:205 overruns:0 frame:0
          TX packets:225 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1000
          RX bytes:153111 (149.5 KiB)  TX bytes:27850 (27.1 KiB)

tc@box:~$ ./rploader.sh update now

Checking Internet Access -> OK
Checking if a newer version exists on the main repo -> sha256sum: latestrploader 
.sh: No such file or directory
Version is current

tc@box:~$ ./rploader.sh fullupgrade now

Performing a full TCRP upgrade
Cloning into 'tinycore-redpill'...
fatal: unable to access 'https://github.com/pocopico/tinycore-redpill.git/': Failed to connect to github.com port 443: No route to host
./rploader.sh: line 1579: cd: /home/tc/tinycore-redpill: No such file or directory
Please make sure you are using the latest 1GB img before using backup option
Current /home/tc size is 49M , try to keep it less than 1GB as it might not fit into your image
Should i update the sda with your current files [Yy/Nn]

always getting no route to host from that points and forwards
i must be dumb!
ty

**Comment by pocopico** [comment]
You can use the control panel on the desktop to set a static IP and gateway if DHCP is not working as expected

**Reply by s0rcier911** [reply]
it is set as static, with a valid IP and valid DNS

**Reply by s0rcier911** [reply]
switching from bridge to NAT seems to works for now, but that is not the way i was looking to make it works.

**Reply by s0rcier911** [reply]
i use a different nic card, and everything start working as intended! in bridged mode ty pocopico!
